src/pages/homeCollection/printLabels.js

```diff
@@ -32,7 +31,7 @@ const printLabelsTemplate = (name) => {
                         <div style="text-align: center;  padding-bottom: 25px; "> 
                           <input required type="text" name="numberToPrint" id="numberToPrint"  /> 
                         </div>
-                        <span> Labels to print: ${ appState.getState().length || 0}  </span>
+                        <span> Labels to print: ${ appState.getState().length || 0 }  </span>
```

Comment:
It's unclear what `length` means here, by only reading this line. I suggest putting `length` under a separate slice of data (eg `labelData`) instead of at root of state data.